index.html

```diff
@@ -0,0 +1,117 @@
+            <div class="section-card">
+                <img  class="sectionImg" src="/images/card1.png" alt="sectionImg1">
+                <div class="section-text card1">
+                    <p class="section-text-top">Hot item</p>
+                    <p class="section-text-middle">인기 상품을<br>확인해 보세요</p>
+                    <p class="section-text-bottom">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</p>
+                </div>
+                
+            </div>
+        </div>
+
+        <div class="section-content">
+            <div class="section-card">
+                <div class="section-text card2">
+                    <p class="section-text-top">Search</p>
+                    <p class="section-text-middle">구매를 원하는<br>상품을 검색하세요</p>
+                    <p class="section-text-bottom">구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</p>
+                </div>
+
+                <img  class="sectionImg" src="/images/card2.png"alt="sectionImg2" >
+                
+            </div>
+        </div>
+
+        <div class="section-content">
+            <div class="section-card card3">
+                <img  class="sectionImg" src="/images/card3.png" alt="sectionImg3" >
+                <div class="section-text">
+                    <p class="section-text-top">Register</p>
+                    <p class="section-text-middle">판매를 원하는<br>상품을 등록하세요</p>
+                    <p class="section-text-bottom">어떤 물건이든 판매하고싶은 상품을<br>쉽게 등록하세요</p>
+                </div>
+                
+            </div>
+        </div>
+       
+    </section>
+<!-- 하단 랜딩 -->
+    <div class="bottomLanding">
+        <div class="bottom-content">
+            <div class="bottom-content-text">
+                <h2>믿을 수 있는<br>판다마켓 중고 거래</h2>
+              
+            </div>
+           
+            <img class="bottomLandingImg" src="/images/bottomImg.png" alt="bottomImg">
+
+        </div>
+    </div>
+<!-- 푸터 -->
+    <footer class="footer">
+        <div class="footer-content">
+
+            <div class="footer-left-text">
+                <p>@codeit - 2024</p>
+            </div>
+            <div class="footer-middle-text">
+                <a href="/pages/privacy.html">Privacy Policy</a>
+                <a href="/pages/faq.html">FAQ</a>
+            </div>
+            <div class="footer-icons">
+                <a href="https://www.facebook.com/" target="_blank">
+                    <img src="/images/icons/ic_facebook.png" alt="facebook">
+                </a>
+                <a href="https://x.com/" target="_blank">
+                    <img src="/images/icons/ic_twitter.png" alt="twitter">
+                </a>
+                <a href="https://www.youtube.com/" target="_blank">
+                    <img src="/images/icons/ic_youtube.png" alt="youtube">
+                </a>
+                <a href="https://www.instagram.com/" target="_blank">
+                    <img src="/images/icons/ic_instagram.png" alt="instagram">
+                </a>
+            </div>
```

Comment:
💊 제안
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
            <ul class="footer-icons">
                <li>
                    <a href="https://www.facebook.com/" target="_blank">
                        <img src="/images/icons/ic_facebook.png" alt="facebook">
                    </a>
                </li>
            </ul>
```